compiler-rt/cmake/builtin-config-ix.cmake

```diff
@@ -162,6 +164,11 @@ if(APPLE)
       list(APPEND DARWIN_tvossim_BUILTIN_ALL_POSSIBLE_ARCHS arm64)
     endif()
   endif()
+  if(COMPILER_RT_ENABLE_XROS)
+    list(APPEND DARWIN_EMBEDDED_PLATFORMS xros)
+    set(DARWIN_xros_BUILTIN_ALL_POSSIBLE_ARCHS arm64)
+    set(DARWIN_xrossim_BUILTIN_ALL_POSSIBLE_ARCHS ${X86_64} arm64)
```

Comment:
(Is there an xros simulator x86_64? I don't have an intel machine to test at hand – just curious.)

Reply:
Per comment below, no x86_64 (I think there was in early betas). Removed.